zegami_sdk/collection.py

```diff
@@ -345,6 +357,40 @@ def download_single(index, url):
             ordered.append(images[i])
         return ordered
 
+    def upload_image(self, path):
+        mime_type = ''
+        file_name = os.path.basename(path)
+        file_ext = os.path.splitext(path)[-1]
+        # Check if the file has a correct extension
+        if file_ext in self.IMAGE_MIMES.keys():
+            mime_type = self.IMAGE_MIMES[file_ext]
+        else:
+            raise ValueError(f"File extension must one of these: {' '.join([key for key in self.IMAGE_MIMES.keys()])}")
+
+        zeg_client = self.client
```

Comment:
Not sure it's worthwhile to assign this - it's only one less character! - and possibly introduces some confusion.